crypto/cms/cms_lib.c

```diff
@@ -431,6 +433,20 @@ BIO *ossl_cms_DigestAlgorithm_init_bio(X509_ALGOR *digestAlgorithm,
         ERR_raise(ERR_LIB_CMS, CMS_R_MD_BIO_INIT_ERROR);
         goto err;
     }
+    if (EVP_MD_is_a(digest, SN_shake128))
+        xof_len = 32;
+    else if (EVP_MD_is_a(digest, SN_shake256))
+        xof_len = 64;
+    if (xof_len > 0) {
+        EVP_MD_CTX *mdctx;
+        OSSL_PARAM params[2];
+        BIO_get_md_ctx(mdbio, &mdctx);
```

Comment:
Please add an empty line after declarations. And also please check return value of BIO_get_md_ctx() and that `mdctx != NULL`.